src/Cli/dotnet/commands/dotnet-workload/install/FileBasedInstaller.cs

```diff
@@ -514,6 +514,7 @@ public void UpdateInstallMode(SdkFeatureBand sdkFeatureBand, bool newMode)
             var installStateContents = InstallStateContents.FromPath(path);
             installStateContents.UseWorkloadSets = newMode;
             File.WriteAllText(path, installStateContents.ToString());
+            _reporter.WriteLine(string.Format(LocalizableStrings.UpdatedWorkloadMode, newMode ? "workload sets" : "loose manifests"));
```

Comment:
If the user runs `dotnet workload update --version 8.0.303`, it's clear that they do want the workload set, but I feel like changing it without saying anything would be an unexpected side-effect. I could imagine someone expecting to update to that workload set but remain in loose manifest mode, as that's the 'minimal' change. I guess what I'm really saying is that although I agree it probably isn't necessary, I don't think it really hurts.

You're right that it won't be localized, but looking at it now, I'm wondering if that might actually be a good thing since the parameter the user will set on the command line isn't localized. I didn't make that here (extra space), but I'm wondering if the best answer might be to make it align with the parameter the user has to pass?

Reply:
> You're right that it won't be localized, but looking at it now, I'm wondering if that might actually be a good thing since the parameter the user will set on the command line isn't localized. I didn't make that here (extra space), but I'm wondering if the best answer might be to make it align with the parameter the user has to pass?

Yes, that's what I was trying to say.  It's OK if it's not localized, but it should match the actual command-line string, and you can use the `WorkloadConfigCommandParser` constants I mentioned to make sure that it's actually the right string.